omniduct/databases/schemas.py

```diff
@@ -45,15 +46,48 @@ def get_columns(self, connection, table_name, schema=None, **kw):
 PrestoDialect.get_columns = get_columns
 
 
+class SchemasMixin(object):
+
+    @property
+    def schemas(self):
+        """
+        This object has as attributes the schemas on the current catalog. These
+        schema objects in turn have the tables as SQLAlchemy `Table` objects.
+        This allows tab completion and exploration of Databases.
+        """
+        from werkzeug import LocalProxy
+
+        def get_schemas():
+            if not getattr(self, '_schemas', None):
+                self.connect()
+                assert getattr(self, '_sqlalchemy_metadata', None) is not None, (
+                    'database must provide sqlalchemy metadata to access schemas'
+                )
+                try:
+                    self._schemas = Schemas(self._sqlalchemy_metadata)
+                except ImportError:
```

Comment:
This error no longer makes sense with this patch, and perhaps never did :). Feel free to remove it.